### PR TITLE
Better logging

### PR DESCRIPTION
**Pull request by roberttod**
- Support a loglevel option to choose what level to log
- Do not log large objects so that the log is more readeable and succinct

### CODE REVIEW

**Comment by roberttod** [comment]
@davidpadbury

**Comment by davidpadbury** [comment]
:+1:

File: lib/log.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+var _ = require('underscore')
+
+createLog.levels = ['info', 'warn', 'error', 'silent']
+module.exports = createLog
```
Comment (jhollingworth) [inline lib/log.js]: personal preference: this feels a bit random at top

File: lib/queue-that.js [hunk 2]
```diff
@@ -51,15 +56,13 @@ function createQueueThat (options) {
...
     storageAdapter.setQueue(options.trim(queue))
 
-    if (activeQueueRunning()) {
-      log('Item(s) sent to active queue')
-      return
-    }
-    switchActiveQueue(queueId)
+    log.info('Item queued')
+
+    if (!activeQueueRunning()) switchActiveQueue(queueId)
```
Comment (jhollingworth) [inline lib/queue-that.js]: You forgot the braces around your if statement

File: lib/log.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+var _ = require('underscore')
```
Comment (jhollingworth) [inline lib/log.js]: why not have this as a library?

Comment (jhollingworth) [inline lib/log.js]: ignore

**Pull request merged**